src/user/tipsandtricks.rst

```diff
@@ -58,7 +58,7 @@ Here is how a ``.condarc`` file would look like:
       - conda-forge
       - defaults
 
-In addition to the channel priority, we recommend to always install your packages inside a new environment instead of the root environment from anaconda/miniconda.
+In addition to the channel priority, we recommend always installing your packages inside a new environment instead of the ``base`` environment from anaconda/miniconda.
```

Comment:
I'm afraid we may still have "root" in some places. But I wonder if we should have a note for old conda users that base used to be root. Maybe not... Too confusing.